beacon-chain/rpc/eth/events/events.go

```diff
@@ -189,15 +198,7 @@ type eventStreamer struct {
 	keepAlive time.Duration
 }
 
-func (es *eventStreamer) streamEvents(ctx context.Context, w StreamingResponseWriter, req *topicRequest, s *Server) error {
```

Comment:
This was a little easier to read originally, I see that the recv Event loop is swapped with the outboxWriteLoop as well.

seems like this wrapper would be a nice to have IMO